src/classes/HelloPortland.cls

```diff
@@ -2,4 +2,10 @@ public with sharing class HelloPortland {
 	public static String sayHello(String name) {
 		return String.format('Hello {0} !', new List<String> {name});
 	}
+
+	public static void Bad_Thing(Integer i) {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.